### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by inkuzmin** [inkuzmin/elm-multiselect#22]
Accessibility is the top priority. The component supports keyboard control quite well but still lacks of ARIA attributes and clear indication of focus. Contributions to change this are most welcome and will be very appreciated! Please, use [react-select](https://jedwatson.github.io/react-select/) as a reference.

**Comment by tryzniak** [comment]
I've been playing around with react-select and noticed that it's only accessibility-enabled for English speaking users. It would be great if `elm-multiselect` was language agnostic in that regard.

**Comment by inkuzmin** [comment]
@tryzniak, it would be absolutely great indeed!
